examples/sync-api/README.md

```diff
@@ -0,0 +1,195 @@
+# TL;DR
+
+If you want to form a relationship with a `_key`, you must include the `_source`
+and `_scope` of the entity that already exists in the graph.
+
+Assuming that the entity you are uploading is the `to` and the entity that you
+are forming a relationship already exists in the graph and therefore is the
+`from`:
+
+This will _NOT_ work:
+
+```
+{
+    _fromEntityKey: entityFrom.entity._key,
+    _toEntityKey: entityTo.entity._key,
+}
+```
+
+This will work:
+
+```
+{
+    _fromEntitySource: entityFrom.entity._source,
+    _fromEntityScope: entityFrom.entity._integrationInstanceId,
+    _fromEntityKey: entityFrom.entity._key,
+    _toEntityKey: entityTo.entity._key,
+}
+```
+
+# Creating Relationship Between Entities You Own With Entities You Do Not
+
+Our goal here is to create relationships from one entity to another that you
+might not own. Ownership is a concept in JupiterOne
```

Comment:
Was there supposed to be more to this sentence? 🤔

Reply:
lmao yes :sadge: I wanted to release it. I'll fix.